### PR TITLE
[DDAPP-12617] revert force phone speaker and update build.gradle

### PR DESCRIPTION
**Pull request by johanvanderboom-tomtom**
Reverting the changes made for forcing playback over only the phone speaker.
Updated build.gradle to be in line with the tts version

### CODE REVIEW

File: android/src/main/java/com/zmxv/RNSound/RNSoundModule.java [hunk 1]
```diff
@@ -1,18 +1,15 @@
...
 import android.content.Context;
 import android.content.res.AssetFileDescriptor;
-import android.media.AudioDeviceInfo;
 import android.media.AudioManager;
 import android.media.AudioAttributes;
 import android.media.AudioFocusRequest;
 import android.media.MediaPlayer;
 import android.media.MediaPlayer.OnCompletionListener;
 import android.media.MediaPlayer.OnErrorListener;
-import android.os.Build.VERSION;
```
Comment (casparvanderheiden-tomtom) [inline android/src/main/java/com/zmxv/RNSound/RNSoundModule.java:15]: Why did you change this import? If we want to merge a newer version or RNSound into our fork, changes like this add unnecessary complexity

File: android/build.gradle [hunk 2]
```diff
@@ -1,39 +1,40 @@
 def safeExtGet(prop, fallback) {
-  rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
```
Comment (casparvanderheiden-tomtom) [inline android/build.gradle:2]: Can you reverse all the whitespace changes (and update your editor), to make it easier when we want to merge the latest version of RNSound into our fork

**Review by casparvanderheiden-tomtom — APPROVED** [review]

File: android/src/main/java/com/zmxv/RNSound/RNSoundModule.java (hunk 1 above)
Comment (johanvanderboom-tomtom) [reply android/src/main/java/com/zmxv/RNSound/RNSoundModule.java:15]: It used to be like `import android.os.Build;` before, I could not get it compiled with the change to `import android.os.Build.VERSION;` so changed it back.

Comment (casparvanderheiden-tomtom) [reply android/src/main/java/com/zmxv/RNSound/RNSoundModule.java:15]: Clear

File: android/build.gradle (hunk 2 above)
Comment (johanvanderboom-tomtom) [reply android/build.gradle:2]: The space changes are not due to my editor, I made a copy of the version used in react-native-tts (which uses proper spacing (4) for gradle files).

I would prefer my version due to the following (but if you still did agree I will change them):
The section in lines 1–14 does not exist in the original and was added in the previous change with wrong spacing [PR](https://github.com/tomtom-international/react-native-sound/pull/7).
The spaces in lines 19 and 20 before that PR are in the original as in my change (previous change changed them).
The spaces in lines 33 and 35 are an indentation fix of something not present in the original repo but added in previous change.
The spaces on lines 22–27 and 39 are added due to this change but I think that might be preferred to get proper spacing for gradle files as only lines 22, 25, 26, 27 are without other changes.

Comment (casparvanderheiden-tomtom) [reply android/build.gradle:2]: Okay, clear
